doc/1-intro/README.md

```diff
@@ -6,6 +6,10 @@ it has a large selection of features and sound chips. from the NES, SNES and Gen
 
 every chip is emulated using many emulation cores, therefore the sound that Furnace produces is authentic to that of real hardware.
 
+## quick start
+
+if you just want to jump right in and get going, it's recommended to go through the [quick start guide](quickstart.md) first. it won't take long, and it'll get you familiar enough with the program to make sense of the rest of the documentation.
```

Comment:
"it's recommended to" -> "we recommend you"

"it won't take long" Are you sure?

Reply:
"It won't take long"... well, I'm still hoping to get a proper estimate by showing it to someone completely new to tracking to see how long it'll take them. I was gonna ask my spouse but they can't commit the time until next weekend.